src/App.svelte

```diff
@@ -1,31 +1,65 @@
 <script>
 	import { Router, Link, Route } from "svelte-routing";
-	import Home from "./routes/Home.svelte";
 	import Manual from "./routes/Manual.svelte";
 	import Automatic from "./routes/Automatic.svelte";
-	import PreparedSequences from "./routes/PreparedSequences.svelte";
 
 	export let url = "";
+	let link = 0;
+	$: config_img = link === 0 ? "images/config_icon.png" : "images/config_icon_off.png";
+	$: touch_img = link === 1 ? "images/touch_icon.png" : "images/touch_icon_off.png";
+	$: config_style = link === 0 ? "color: black;" : "color: gray;";
+	$: touch_style = link === 1 ? "color: black;" : "color: gray;";
+
```

Comment:
Extra svelte-y pattern would be to pull this into a component, then bind these as `{src}` and `{style}`